packages/js/package.json

```diff
@@ -16,12 +16,16 @@
       "default": "./dist/esm/index.mjs"
     },
     "./ui": {
-      "types": "./dist/ui/index.d.ts",
-      "require": "./dist/ui/index.js",
-      "import": "./dist/ui/index.mjs",
-      "default": "./dist/ui/index.mjs"
+      "types": "./dist/types/ui/index.d.ts",
+      "require": "./dist/cjs/ui/index.cjs",
+      "import": "./dist/esm/ui/index.mjs",
+      "default": "./dist/esm/ui/index.mjs"
     },
-    "./ui/index.css": "./dist/ui/index.css"
+    "./ui/index.css": {
+      "require": "./dist/cjs/ui/index.css",
+      "import": "./dist/esm/ui/index.css",
+      "default": "./dist/esm/ui/index.css"
+    }
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** This can be removed as you're using .directcss now‏